### PR TITLE
Add pod basic test case

### PR DESCRIPTION
**Pull request by xlgao-zju**
Signed-off-by: Xianglin Gao <xlgao@zju.edu.cn>

### CODE REVIEW

**Comment by xlgao-zju** [comment]
@feiskyer @Random-Liu

File: pkg/validate/pod.go [hunk 1]
```diff
@@ -32,7 +33,46 @@ var _ = framework.KubeDescribe("Test CRI PodSandbox", func() {
 		c = f.CRIClient.CRIRuntimeClient
 	})
 
-	It("test get version info", func() {
+	It("test get version info [Conformance]", func() {
```
Comment (feiskyer) [inline pkg/validate/pod.go]: We should make this a complete sentence (and all followings), e.g. `runtime should return version info`. And `Version()` should be outside of sandbox context.

File: pkg/framework/util.go [hunk 2]
```diff
@@ -105,7 +109,111 @@ func NewUUID() string {
 
 // TestGetVersion test if we can get runtime name.
 func TestGetVersion(c internalapi.RuntimeService) {
-	version, err := c.Version(defaultAPIVersion)
+	_, err := c.Version(defaultAPIVersion)
```
Comment (feiskyer) [inline pkg/framework/util.go]: Should check the output, e.g. all response strings should not be null.

File: pkg/framework/util.go [hunk 3]
```diff
@@ -105,7 +109,111 @@ func NewUUID() string {
...
+	ExpectNoError(err, "failed to create PodSandbox: %v", err)
+	Logf("Created PodSandbox %q\n", podID)
+	return podID
+}
+
+// RunDefaultPodSandbox runs a PodSandbox with default options.
+func RunDefaultPodSandbox(c internalapi.RuntimeService, prefix string) string {
+	podSandboxName := prefix + NewUUID()
+	config := &runtimeapi.PodSandboxConfig{
+		Metadata: buildPodSandboxMetadata(podSandboxName, defaultUID, defaultNamespace, defaultAttempt),
```
Comment (feiskyer) [inline pkg/framework/util.go]: Add random to sandbox ID/name/namespace, so the test cases could be easily paralleled later.

**Comment by xlgao-zju** [comment]
@feiskyer updated

**Comment by feiskyer** [comment]
LGTM.

**Pull request merged**